src/test/regress/expected/citus_cdc_basic_distributed_table_concurrently.out

```diff
@@ -0,0 +1,236 @@
+/*
+Citus CDC basic Test to verify a distributed table's INSERT/UPDATE/DELETE operations are published correctly to a subscriber without duplication.
```

Comment:
this file does not have .sql   and not added to any schedules? Similar for citus_cdc_basic_distributed_table

Reply:
yes I had it before but removed it to just have one way of test cases with TAP tests.
I will remove this file.

Reply:
> I will remove this file.

please also remove `citus_cdc_basic_distributed_table ` which seems to have a similar problem

Reply:
I have removed both now